src/command/file/file-command.ts

```diff
@@ -7,7 +7,6 @@ export class FileCommand extends RootCommand {
     key: 'account',
     alias: 'a',
     description: 'Name of the account',
-    required: { when: 'quiet' },
```

Comment:
keep quiet mode for non-interactive sessions, because of this, leave it here for now.

Reply:
same at `pod`